fuzz/src/bin/gen_target.sh

```diff
@@ -70,3 +70,8 @@ GEN_TEST msg_tx_signatures msg_targets::
 GEN_TEST msg_tx_init_rbf msg_targets::
 GEN_TEST msg_tx_ack_rbf msg_targets::
 GEN_TEST msg_tx_abort msg_targets::
+
+GEN_TEST msg_stfu msg_targets::
```

Comment:
nit: I guess this is technically independent of splicing so should stand in its own group but no big deal

Reply:
I was unsure, as `stfu` is being added to the spec with the same PR. But let's keep separate, I reformatted in light of this.